src/content-helper/editor-sidebar/performance-details/component.tsx

```diff
@@ -73,233 +232,79 @@ export function PerformanceDetails(
 		};
 	}, [ period ] );
 
-	if ( error ) {
-		return error.Message();
-	}
-
 	return (
-		loading
-			?	(
-				<div className="parsely-spinner-wrapper" data-testid="parsely-spinner-wrapper">
-					<Spinner />
+		<div className="wp-parsely-performance-panel">
+			<PerformanceStatPanel
+				title={ __( 'Performance Stats', 'wp-parsely' ) }
+				icon={ moreVertical }
+				dropdownChildren={ ( { onClose } ) => <PerformanceStatsMenu onClose={ onClose } /> }
+			>
+				<div className="panel-settings">
+					<SelectControl
+						value={ settings.PerformanceStatsSettings.Period }
+						prefix={ __( 'Period: ', 'wp-parsely' ) }
```

Comment:
@acicovic @vaurdan they provide a <InputControlPrefixWrapper> component to wrap the prefix and <InputControlSuffixWrapper> to wrap the suffix. That adds the correct padding when the component has a prefix or suffix added. https://github.com/WordPress/gutenberg/pull/59719 I opened a PR but we can probably implement it the same way here.

Reply:
InputControlPrefixWrapper and InputControlSuffixWrapper are the ones you want.

Reply:
There also seem to be a bunch of size options that don't exist in Figma. We want the one that's 40px hight. I think it's called __unstable-large

Reply:
Thanks for looking! I will try to implement the prefix and sufix components and will open a PR once that's done.

Reply:
Addressed in https://github.com/Parsely/wp-parsely/pull/2278